### PR TITLE
add departments to drop down list

### PR DESCRIPTION
**Pull request by Troy-Lawson**
add cirnac and parliamentary protection to list and remove inac

### CODE REVIEW

File: mod/c_members_byDepartment/pages/members/members_by_dept.php [hunk 1]
```diff
@@ -46,16 +47,17 @@
...
 $provinces['pov-nb'] = 'Gouvernement du Nouveau-Brunswick';
 $provinces['pov-nfl'] = 'Gouvernement de Terre-Neuve-et-Labrador';
-$provinces['pov-ns'] = 'Gouvernement de la Nouvelle-�cosse';
+$provinces['pov-ns'] = 'Gouvernement de la Nouvelle-�cosse';
 $provinces['pov-nwt'] = 'Gouvernement du Territoires du Nord-Ouest';
 $provinces['pov-nun'] = 'Gouvernement du Nunavut';
 $provinces['pov-ont'] = "Gouvernement de l'Ontario";
-$provinces['pov-pei'] = "Gouvernement de l'�le-du-Prince-�douard";
-$provinces['pov-que'] = 'Gouvernement du Qu�bec';
+$provinces['pov-pei'] = "Gouvernement de l'�le-du-Prince-�douard";
```
Comment (ethanWallace) [inline mod/c_members_byDepartment/pages/members/members_by_dept.php]: These � characters show up in lists.

File: mod/c_members_byDepartment/pages/members/members_by_dept.php [hunk 2]
```diff
@@ -37,7 +37,8 @@
 $provinces['pov-sask'] = 'Government of Saskatchewan';
 $provinces['pov-yuk'] = 'Government of Yukon';
 $departmentsEn = array_merge($departmentsEn,$provinces);
-
+unset($departmentsEn['ou=INAC-AANC, o=GC, c=CA']);
```
Comment (ethanWallace) [inline mod/c_members_byDepartment/pages/members/members_by_dept.php]: Is there a reason we are not adding the two new departments to this page?

File: mod/c_members_byDepartment/pages/members/members_by_dept.php (hunk 1 above)
Comment (Troy-Lawson) [reply mod/c_members_byDepartment/pages/members/members_by_dept.php]: Not sure why this happened but fixed it where it did

File: mod/c_members_byDepartment/pages/members/members_by_dept.php (hunk 2 above)
Comment (Troy-Lawson) [reply mod/c_members_byDepartment/pages/members/members_by_dept.php]: Missed it on this page. Added them in

File: mod/c_members_byDepartment/pages/members/index.php [hunk 3]
```diff
@@ -182,7 +184,9 @@ function render_department_tab($data_directory) {
         $provinces['pov-pei'] = "Gouvernement de l'�le-du-Prince-�douard";
```
Comment (nschonni) [inline mod/c_members_byDepartment/pages/members/index.php]: Few other encoding issues in this one too

File: mod/wet4/views/default/verify_department/verify_department.php [hunk 4]
```diff
@@ -51,9 +53,12 @@
     $provinces['pov-que'] = 'Gouvernement du Qu�bec';
```
Comment (nschonni) [inline mod/wet4/views/default/verify_department/verify_department.php]: Not introduced, but a few more encoding issues

**Pull request merged**